pkg/broker/ingress/ingress_handler.go

```diff
@@ -117,16 +116,11 @@ func (h *Handler) decrementTTL(event *cloudevents.Event) bool {
 	return true
 }
 
-func (h *Handler) getTTLToSet(event *cloudevents.Event) int {
-	ttlInterface, _ := broker.GetTTL(event.Context)
-	if ttlInterface == nil {
-		h.Logger.Debug("No TTL found, defaulting")
+func (h *Handler) getTTLToSet(event *cloudevents.Event) int32 {
+	ttl, err := broker.GetTTL(event.Context)
+	if err != nil {
+		h.Logger.Info("Error retrieving TTL, defaulting.", zap.Error(err))
```

Comment:
This feels like it should be a debug, since it will happen once for every event.

Reply:
Any specific reason why it should be debug?

Reply:
Performance, but it's minor.